lib/DxcSupport/HLSLOptions.cpp

```diff
@@ -446,7 +446,7 @@ int ReadDxcOpts(const OptTable *optionTable, unsigned flagsToInclude,
   } else {
     try {
       opts.HLSLVersion = std::stoul(std::string(ver));
-      if (opts.HLSLVersion < 2015 || opts.HLSLVersion > 2018) {
+      if (opts.HLSLVersion < 2015 || opts.HLSLVersion > 2021) {
```

Comment:
Maybe we should update the help text for -HV (look for `hlsl_version`) to indicate 2021 is a valid option.  Also, since there are only specific values that are valid (2015, 2016, 2018, 2021), maybe we should use a switch statement with the valid versions here instead of a range?

Reply:
I intended to do the first, but forgot and was considering the second

Reply:
Note: help text doesn't mention 2015 (that's because that's an intellisense-only target), and does mention 2017, even though I don't recall if there is any differentiation for that version.

Reply:
Oh, nevermind, 2017 is valid, enums were introduced there.